core/src/main/scala/org/apache/spark/util/DependencyUtils.scala

```diff
@@ -39,11 +40,11 @@ private[spark] object DependencyUtils extends Logging {
 
   def getIvyProperties(): IvyProperties = {
     val Seq(packagesExclusions, packages, repositories, ivyRepoPath, ivySettingsPath) = Seq(
-      "spark.jars.excludes",
-      "spark.jars.packages",
-      "spark.jars.repositories",
-      "spark.jars.ivy",
-      "spark.jars.ivySettings"
+      config.JAR_PACKAGES_EXCLUSIONS.key,
```

Comment:
import config._ and remove the config. part from here

Reply:
Done